### DISCUSSION TITLE
Google, Facebook login on 3.0

### DISCUSSION THREAD

**Discussion by edsdck** [saleor/saleor#8177] [Q&A] [answered]
Hello, is there going to be an open-source plugin for login using Google or other third-party services on Saleor 3.0?

**Comment by karolkielecki** [comment, marked as answer]
@edsdck Hey, the OIDC plugin is available but only on Saleor Cloud--as stated in our documentation. You can create your own plugin using OIDC.

**Reply by edsdck** [reply]
Thank you for your response.

And you don't have any plans to release it publicly, am I right?

**Reply by karolkielecki** [reply]
There is no plan to release it. If there are any changes, I'll keep you posted.

**Reply by Yahyaali1** [reply]
Hello 
I see that we have support of OIDC on dashboard. Can that be used to implement google sign in for frontend app ?

**Reply by fedexin40** [reply]
@Yahyaali1 I think the answer is yes, I already did it :100:

**Reply by Yahyaali1** [reply]
@fedexin40 can you share your approach ?
